### DISCUSSION TITLE
请问哪里可以订阅开发进度呢？

### DISCUSSION THREAD

**Discussion by Rainxbow** [ByteLegend/ByteLegend#72] [Q&A]
基本养成了每天上来看一眼的习惯了。

**Comment by gtn1024** [comment]
看这个仓库有没有[ByteLegendBot](https://github.com/ByteLegendBot)的`release`提交

![image](https://user-images.githubusercontent.com/69113276/135480182-799f9830-d9ce-4bae-a22d-7e13079a39b0.png)

**Reply by blindpirate** [reply]
不是的，release操作只是打包，并不一定部署，要看deploy是否有更新：https://github.com/ByteLegend/ByteLegend/deployments/activity_log?environment=production

**Reply by blindpirate** [reply]
承蒙厚爱，正在写一个批量的仓库管理系统，完事后就可以把 https://github.com/hcsp 的200个挑战的题目迁移过来了，后续进度会加快。
